packages/rollup-config/eslint.config.js

```diff
@@ -14,8 +14,6 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-const TOKEN = "secret"
+import base from "@solarwinds-apm/eslint-config"
 
-module.exports = {
-  TOKEN,
-}
```

Comment:
This wasn't used in the example anymore and I guess git decided I actually moved the file and changed everything in it.